### ISSUE TITLE
[lhBrackets] Typing the closing character should delete the placeholder that follows (if any)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/lh-vim#10]
```
What steps will reproduce the problem?
- type (foo)

What is the expected output? 
 (foo)<cursor>

What do you see instead?
 (foo)<cursor><+placeholder+>





```

Original issue reported on code.google.com by `luc.herm...@gmail.com` on 22 Oct 2008 at 11:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `luc.herm...@gmail.com` on 23 Oct 2008 at 3:23
- Added labels: **lhBrackets**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r264 (lh-brackets v 1.1.0)
```

Original comment by `luc.herm...@gmail.com` on 1 Dec 2010 at 9:02
- Changed state: **Fixed**
